lb.tf

```diff
@@ -0,0 +1,55 @@
+resource "aws_lb" "main" {
+  count = var.nlb_enabled ? 1 : 0
```

Comment:
it should enable is also when `var.deployment_mode` is set to "ACTIVE_STANDBY_MULTI_AZ"